src/components/Elements/Balance/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography, useTheme } from '@mui/material';
+
+import { formatBalance } from '@/utils/functions';
+
+interface BalanceProps {
+  coretimeBalance: number;
+  relayBalance: number;
+  symbol: string;
+}
+
+const Balance = ({ relayBalance, coretimeBalance, symbol }: BalanceProps) => {
+  const theme = useTheme();
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <Typography sx={{ color: theme.palette.text.primary, my: '0.5em' }}>
+        {`Relay chain: ${formatBalance(relayBalance.toString(), false)} ${symbol}`}
```

Comment:
use `relayChainSymbol`

Reply:
Actually, the symbol on both of these chains will always be the same since the Coretime chain is a system chain, so they use the same currency.